### PR TITLE
signup-login: override layout and styling

### PR DESCRIPTION
**Pull request by jennur**
Closes https://github.com/zenodo/zenodo-rdm/issues/62

Screenshot 2022-10-28 at 10 45 22
Screenshot 2022-10-28 at 10 45 14

### CODE REVIEW

File: assets/less/zenodo-rdm/collections/form.overrides [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+form[name="register_user_form"],
```
Comment (kpsherva) [inline assets/less/zenodo-rdm/collections/form.overrides:2]: is there any other way we could match these elements, fe by class name ? attribute selectors tend to be less efficient

File: assets/less/zenodo-rdm/elements/segment.overrides [hunk 2]
```diff
@@ -28,4 +28,16 @@
       padding: 0;
     }
   }
+}
+
+.ui.segments.very.rounded {
```
Comment (kpsherva) [inline assets/less/zenodo-rdm/elements/segment.overrides:33]: are you trying to decrease the "roundiness" of the element? in this case wouldn't it be enough to remove "very" class?

Comment (jennur) [reply assets/less/zenodo-rdm/elements/segment.overrides:33]: no, the opposite 😄 I'm increasing it, and tried to match the equivalent class-naming for padded segments (`.ui.padded.segment` & `.ui.very.padded.segment`)

File: assets/less/zenodo-rdm/elements/divider.overrides [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
...JlYWR5ccllPAAAAyFpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuNS1jMDE0IDc5LjE1MTQ4MSwgMjAxMy8wMy8xMy0xMjowOToxNSAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENDIChXaW5kb3dzKSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDo1OThBRDY4OUNDMTYxMUU0OUE3NUVGOEJDMzMzMjE2NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDo1OThBRDY4QUNDMTYxMUU0OUE3NUVGOEJDMzMzMjE2NyI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOjU5OEFENjg3Q0MxNjExRTQ5QTc1RUY4QkMzMzMyMTY3IiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU5OEFENjg4Q0MxNjExRTQ5QTc1RUY4QkMzMzMyMTY3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+VU513gAAADVJREFUeNrs0DENACAQBDBIWLGBJQby/mUcJn5sJXQmOQMAAAAAAJqt+2prAAAAAACg2xdgANk6BEVuJgyMAAAAAElFTkSuQmCC);
```
Comment (kpsherva) [inline assets/less/zenodo-rdm/elements/divider.overrides:10]: I am not sure why do we include an image in this fashion here? Which image is it?

Comment (slint) [reply assets/less/zenodo-rdm/elements/divider.overrides:10]: I think it's the search results item divider (which indeed is weird). Doesn't an `<hr>` element or a SUI equivalent work?

Comment (jennur) [reply assets/less/zenodo-rdm/elements/divider.overrides:10]: It's for the "OR" divider. There is something funky going on that probably should be investigated, the horizontal/vertical divider in SUI https://semantic-ui.com/elements/divider.html has this bg-image, but it doesn't in our theme, so there is no divider, just the "OR" text.

**Review by slint — APPROVED** [review]
LGTM. I've added some comments on the copyright headers, but maybe we tackle all of these together in a separate PR, since it's already inconsistent throughout the repo and a decision has to be made.

**Pull request merged**